libs.versions.toml

```diff
@@ -0,0 +1,161 @@
+# Also see settings.gradle file for dynamic versionCatalogs dependencies
+
+[versions]
+antlrVersion = "4.8"
+awsSdkVersion = "2.20.18"
+flink115Version = "1.15.0"
+flink116Version = "1.16.2"
+flink117Version = "1.17.1"
+hadoop2Version = "2.7.3"
+hive2Version = "2.3.9"
```

Comment:
Might be necessary to declare a version-range for Hive 2 to "not confuse Dependabot" (see below / Jackson comment). Sad that neither Hive nor Hadoop publish a BOM.

Reply:
done